File: PYTHON/Colleen.py
# hello program

def print_str(x):
    print(x.format(chr(10),chr(32)*4,chr(34),x))

if __name__ == "__main__":
    # hello main
    x = "# hello program{0}{0}def print_str(x):{0}{1}print(x.format(chr(10),chr(32)*4,chr(34),x)){0}{0}if __name__ == {2}__main__{2}:{0}{1}# hello main{0}{1}x = {2}{3}{2};print_str(x)";print_str(x)

File: C/Colleen.c
#include <stdio.h>
/*
	Hello program.
*/
void print(char *s)
{
	printf(s,10,10,9,10,10,10,10,9,10,10,10,10,9,10,9,9,10,9,10,9,34,s,34,10,9,10,9,10,10);
}
int main()
{
	/*
		Hello main.
	*/
	char *s = "#include <stdio.h>%c/*%c%cHello program.%c*/%cvoid print(char *s)%c{%c%cprintf(s,10,10,9,10,10,10,10,9,10,10,10,10,9,10,9,9,10,9,10,9,34,s,34,10,9,10,9,10,10);%c}%cint main()%c{%c%c/*%c%c%cHello main.%c%c*/%c%cchar *s = %c%s%c;%c%cprint(s);%c%creturn (0);%c}%c";
	print(s);
	return (0);
}
